partials/content-cover-book-header.php

```diff
@@ -72,7 +72,7 @@
 						// }, 10, 2); @codingStandardsIgnoreEnd
 						$tracking = apply_filters( 'pressbooks_download_tracking_code', "ga('send','event','exportFiles','Downloads','{$title}:{$filetype}');", $filetype, $title );
 						?>
-					<li class="dropdown-item">
+					<li class="dropdown-item" tabindex="0">
```

Comment:
I wouldn't advise this as it adds non-interactive elements to the tab order. Now, as a screen reader user, I can focus both the list item and the link contained within it. For example, VoiceOver on macOS would read out: `EPUB, link, group` and immediately after: `EPUB, link`.